server/data/prisonerContactRegistryApiClient.ts

```diff
@@ -24,7 +24,7 @@ class PrisonerContactRegistryApiClient {
         }).toString(),
       })
     } catch (e) {
-      if (e.data.status !== 404) {
+      if (e.status !== 404) {
```

Comment:
Unrelated, but this was also causing failures because `status` is on the response `error` object, not `error.data`